account_balance_query.go

```diff
@@ -312,3 +312,12 @@ func (query *AccountBalanceQuery) SetPaymentTransactionID(transactionID Transact
 	query.paymentTransactionIDs._Clear()._Push(transactionID)._SetLocked(true)
 	return query
 }
+
+func (query *AccountBalanceQuery) SetLogLevel(level LogLevel) *AccountBalanceQuery {
+	query.Query.logLevel = &level
+	return query
+}
+
+func (query *AccountBalanceQuery) GetLogLevel() *LogLevel {
+	return query.Query.logLevel
+}
```

Comment:
It would probably be better if all query and transactions use the base Query/Transaction methods for setting/getting log level.

Reply:
I will leave the `GetLogLevel()` method only in the base Query and Transaction structs as a method ( removed from the types of transactions and queries )

refactoring the `SetLogLevel()` to use the base methods like so:
`
...{
    transaction.Transaction.SetLogLevel(level)
    return transaction
}
`